src/ssl.mli

```diff
@@ -418,7 +418,11 @@ val accept : socket -> unit
 (** Flush an SSL connection. *)
 val flush : socket -> unit
 
-(** Close an SSL connection. *)
+(** send close notify to the peer. This is SSL_shutdown(3) *)
+val close_notify : socket -> int
```

Comment:
The API follows the general convention that the return value is never exposed, because it is quite meaningless in OCaml. A boolean indicating whehter the shutdown is finished or not (and exceptions in other cases) would be better.

Reply:
I changed it to bool.